python-ci/pipeline.yaml

```diff
@@ -0,0 +1,37 @@
+env:
+  PYTHON_VERSION: "3"
+
+steps:
+  - label: ":pip: Install dependencies"
+    key: "pip"
+    command: pip install -r requirements.txt
+    plugins:
+      - docker#v5.9.0:
+          image: "python:${PYTHON_VERSION}"
```

Comment:
What's the preferred docker image version to enter here?

Should we use `latest` or an env variable?

Reply:
Yeah, let's stick with latest.